task/06-conditions-n-loops-tasks.js

```diff
@@ -82,7 +101,10 @@ function getSumBetweenNumbers(n1, n2) {
  *   10,10,10 =>  true
  */
 function isTriangle(a,b,c) {
-    throw new Error('Not implemented');
+     
+    if(a+b>c && a+c>b && b+c>a)
```

Comment:
HERE and BELOW Please, try to follow these rules https://javascript.info/coding-style
(spaces around operators)

https://developer.mozilla.org/en-US/docs/Web/JavaScript/Guide/Expressions_and_Operators